.ruby-version

```diff
@@ -0,0 +1 @@
+1.9.3-p551
```

Comment:
I committed this accidentally. Will remove. We use Travis for this, so tests in multiple rubies